src/mon/OSDMonitor.cc

```diff
@@ -4837,6 +4839,31 @@ void OSDMonitor::check_pg_creates_sub(Subscription *sub)
   }
 }
 
+void OSDMonitor::check_for_filestore_osds(health_check_map_t *checks)
+{
+  set<int32_t> filestore_osds;
+  for (int i = 0; i < (int)osdmap.get_num_osds(); ++i) {
+    string objectstore_type;
+    int r = get_osd_objectstore_type(i, &objectstore_type);
+    if (r == 0 && objectstore_type == "filestore") {
+      filestore_osds.insert(i);
+    }
+  }
+
+  if (filestore_osds.size() > 0) {
+    ostringstream ss;
+    ss << filestore_osds.size() << " osds on filestore";
+    auto& d = checks->add("OSD_FILESTORE", HEALTH_WARN, ss.str(),
```

Comment:
we need to add a config option to suppress this warning for testing

Reply:
We could, but that would mean muting this warning for all tests that use filestore. It would be easier to globally disable this waring for teuthology tests using a separate config.